hive-metastore/src/test/java/org/apache/iceberg/hive/HiveMetastoreTest.java

```diff
@@ -31,6 +31,10 @@
 import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.BeforeAll;
 
+/*
+ * This meta-setup has been deprecated; use {@link HiveMetastoreExtension} instead.
+ * */
+@Deprecated
```

Comment:
After this could you please create a PR to remove this class?
Or in this one, if it is trivial...

Reply:
Thanks @pvary for reviewing this . Yes we have another change is in progress with all the tests related cleanup. 
WIP [patch](https://github.com/nk1506/iceberg/pull/2)

Reply:
@nk1506 can you please open a proper PR against the Iceberg repo with those changes where subclasses of `HiveMetastoreTest` are moved to JUnit5?

Reply:
Created a [PR](https://github.com/apache/iceberg/pull/9282)  with jUnit5.